strongback/src/org/strongback/hardware/Hardware.java

```diff
@@ -781,7 +781,7 @@ public static InputDevice driverStationJoystick(int port) {
          * @param port the port on the driver station that the flight stick is plugged into
          * @return the input device; never null
          */
-        public static FlightStick logitechAttack3D(int port) {
+        public static FlightStick logitechAttack3(int port) {
```

Comment:
The only reason I thought to change it was because with the addition of the Extreme 3D joystick, it may be confusing to have two joysticks with "3D" at the end when referring to them. 

I'm not sure if it's actually worth changing - if you don't think so, I can undo that change. Otherwise, I could add in the annotation as you suggested.